pretix_passbook/passbook.py

```diff
@@ -433,6 +433,18 @@ def generate_pass(self, order_position: OrderPosition):
                 gettext("Website"),
             )
 
+        try:
+            backfieldprop = order_position.item.meta_values.get(property__name="pretix_passbook_backfield")
+
+            if backfieldprop.value:
+                card.addBackField(
+                    "metabackfield",
+                    backfieldprop.value,
```

Comment:
This does not account for the fact that the meta property might have a default value.

Untested suggestion:

```suggestion
            backfieldprop = order_position.item.meta_data.get("pretix_passbook_backfield")

            if backfieldprop:
                card.addBackField(
                    "metabackfield",
                    backfieldprop,
```